client/src/profile/AddressField.js

```diff
@@ -0,0 +1,89 @@
+import React, { PropTypes } from 'react';
+import AutoComplete from 'react-autocomplete';
+// import { reduxForm, Field } from 'redux-form';
+
+const MAPZEN_API_KEY = 'mapzen-Xt7d1Sz';
+const MAPZEN_AUTOCOMPLETE_URL = 'https://search.mapzen.com/v1/autocomplete';
+const SF_LAT = '37.7';
+const SF_LON = '-122.4';
+
+const styles = {
+  item: {
+    padding: '2px 6px',
+    cursor: 'default',
+  },
+
+  highlightedItem: {
+    color: 'white',
+    background: 'hsl(200, 50%, 50%)',
+    padding: '2px 6px',
+    cursor: 'default',
+  },
+};
+
+function getURL(baseURL, getParams) {
```

Comment:
They don't need access to this, though, so I will get an error if I do that, right?